site-src/guides/index.md

```diff
@@ -56,55 +55,136 @@ This quickstart guide is intended for engineers familiar with k8s and model serv
-   ```bash
-   kubectl apply -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/enable_patch_policy.yaml
-   kubectl rollout restart deployment envoy-gateway -n envoy-gateway-system
-   ```
-   Additionally, if you would like to enable the admin interface, you can uncomment the admin lines and run this again.
+   Choose one of the following options to deploy an Inference Gateway.
 
-### Deploy Gateway
+=== "GKE"
 
-   ```bash
-   kubectl apply -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/gateway.yaml
-   ```
-   > **_NOTE:_** This file couples together the gateway infra and the HTTPRoute infra for a convenient, quick startup. Creating additional/different InferencePools on the same gateway will require an additional set of: `Backend`, `HTTPRoute`, the resources included in the `./config/manifests/gateway/ext-proc.yaml` file, and an additional `./config/manifests/gateway/patch_policy.yaml` file. ***Should you choose to experiment, familiarity with xDS and Envoy are very useful.***
+      1. Enable the Gateway API
+
+         ```bash
+         gcloud container clusters update <CLUSTER_NAME> \
+             --location=<CLUSTER_LOCATION> \
+             --gateway-api=standard
+         ```
+
+      1. Create the proxy-only subnet
+      
+         A proxy-only subnet provides a set of IP addresses that Google uses to run Envoy proxies on your behalf. 
+         ```
+         gcloud compute networks subnets create proxy-only-subnet \
+             --purpose=REGIONAL_MANAGED_PROXY \
+             --role=ACTIVE \
+             --region=<REGION> \
+             --network=<VPC_NETWORK_NAME> \
+             --range=<CIDR_RANGE>
+         ```
+
+      1. Deploy Gateway and HealthCheckPolicy resources
+
+         ```bash
+         kubectl apply -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/gke/gateway.yaml
+         kubectl apply -f https://github.com/kubernetes-sigs/gateway-api-inference-extension/raw/main/config/manifests/gateway/gke/healthcheck.yaml
+         ```
+
+         Confirm that the Gateway was assigned an IP address and reports a `Programmed=True` status:
+         ```bash
+         $ kubectl get gateway inference-gateway
+         NAME                CLASS               ADDRESS         PROGRAMMED   AGE
+         inference-gateway   inference-gateway   <MY_ADDRESS>    True         22s
+         ```
+
+=== "Istio"
+
+      Please note that this feature is currently in an experimental phase and is not intended for production use. 
+      The implementation and user experience are subject to changes as we continue to iterate on this project.
+
+      1. Install Istio
+      
+         Please follow the [Istio installation guide](https://istio.io/latest/docs/setup/install/).
```

Comment:
@LiorLieberman don't users need to use a specific branch of Istio to get this functionality?

Reply:
They do, I left that comment above, waiting for a release.

Reply:
after release is there, I would just point to an installation guide with overriding the images tag.